### ISSUE TITLE
the Log example reads entire conversation from the top every time a line is added

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#1186]
https://www.w3.org/WAI/WCAG21/working-examples/aria-role-log/chatlog.html

Every time a line is added it reads the entire conversation. Instead we want it to read only the new content that is added to the conversation.

In Jaws and NVDA when the user clicks send it says:

Hello

Hello
Hi!

Hello
Hi!
Pleased to meet you.

Hello
Hi!
Pleased to meet you.
We'll talk later, okay?

Hello
Hi!
Pleased to meet you.
We'll talk later, okay?
Bye!

We might be able to leverage "additions" to fix it. Otherwise we'll need to put the log on each new div 300ms before the text is inserted which could be a pain. I can test.

**Comment by JAWS-test** [comment]
If I see this correctly it is a bug in the browser and screenreader and not a bug in the WCAG example, because aria-atomic should be false by default for role=log and therefore only the changed content should be output.

**Comment by DavidMacDonald** [comment]
It does this also with aria-live on it instead of role="log"

**Comment by DavidMacDonald** [comment]
PS just tested in Firefox (latest build) 
Works properly on NVDA
Live region not read in JAWS 2020

**Comment by giacomo-petri** [comment]
Tested also with:
- JAWS on Chrome
- VoiceOver on Chrome
- VoiceOver on Safari

and they all behave as expected reading only the specific line.
Closing as no longer reproducible. 

If you believe the issue persists on specific combination of techs feel free to reopen.
